allensdk/brain_observatory/behavior/vbo_2023_nwb_writers/nwb_writer_utils.py

```diff
@@ -0,0 +1,92 @@
+import logging
+from typing import List, Union
+import pytz
+
+from allensdk.brain_observatory.behavior.behavior_ophys_experiment import (
+    BehaviorOphysExperiment,
+)
+from allensdk.brain_observatory.behavior.behavior_session import (
+    BehaviorSession,
+)
+from allensdk.brain_observatory.behavior.data_files.stimulus_file import (
+    BehaviorStimulusFile,
+)
+from allensdk.core.auth_config import LIMS_DB_CREDENTIAL_MAP
+from allensdk.internal.api import db_connection_creator
+
+
+def _compare_metadata(
```

Comment:
Again, this logic is general to all projects which we release data for and should be placed in a general location rather than vbo.